user_stories.md

```diff
@@ -18,7 +19,8 @@
 
 ## 🛍️ Seller User Stories – Digital Goods E-commerce Platform
 
-- Seller can sign up and log in securely using email OTP.
+- Seller can sign up using email and password (no OTP required).
+- Seller can log in securely using verified credentials.
 - Seller can create and manage their profile with bio, avatar, and contact details.
 - Seller can upload new digital products (eBooks, PDFs, art, videos, etc.).
```

Comment:
@sanjeet1999 you need to be very specific here when it comes to product categories being supported here, you cannot mention etc .